recipes/behaviortree.cpp/all/conandata.yml

```diff
@@ -1,28 +1,47 @@
 sources:
+  "4.1.1":
+    url: "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/4.1.1.tar.gz"
+    sha256: "3f009798154964dec662d5708c2d3f2fae224dedf12690be56d47752ca5714af"
+  "4.0.2":
+    url: "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/4.0.2.tar.gz"
+    sha256: "0f93d3460cbc48c57c7522dd78fc04bb7c06bd7dfbd68b2771adfe6ef9174aef"
   "4.0.1":
     url: "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/4.0.1.tar.gz"
     sha256: "71544f72abea8e8c246b016b7e8d87d96f731c8aa96698058d8e69d40e56f9b9"
+  "3.8.3":
+    url: "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/3.8.3.tar.gz"
+    sha256: "5dd782cc73913b0bfe26adb6fe3d7c10f1e2063a0a994bcba4d4e2ac86470196"
   "3.7.0":
     url: "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/3.7.0.tar.gz"
     sha256: "ab0d8ac1a0df4dd43cf45da8a784bab7fdedf711bd0e227f7ed071f79b0c7b5c"
   "3.5.6":
     url: "https://github.com/BehaviorTree/BehaviorTree.CPP/archive/refs/tags/3.5.6.tar.gz"
     sha256: "543c428602b5acb7c5666aee34feb532e18ce7200870a79b23ff9aed17ee84c4"
 patches:
+  "4.1.1":
+    - patch_file: "patches/4.1.1-0001-cmake-version.patch"
+      patch_description: "change minimum cmake version to 3.15.7"
+      patch_type: "conan"
+  "4.0.2":
+    - patch_file: "patches/4.0.1-0001-remove-fpic.patch"
+      patch_description: "remove CMAKE_POSITION_INDEPENDENT_CODE definition"
+      patch_type: "conan"
   "4.0.1":
     - patch_file: "patches/4.0.1-0001-remove-fpic.patch"
-    - patch_file: "patches/4.0.1-0002-find-zmq.patch"
```

Comment:
I am a bit confused about this change, how it is possible to remove patches that were there in the past for an already packaged version?